src/core/a-mixin.js

```diff
@@ -9,106 +9,91 @@ var MULTIPLE_COMPONENT_DELIMITER = '__';
  * @member {object} componentCache - Cache of pre-parsed values. An object where the keys
  *         are component names and the values are already parsed by the component.
  */
-module.exports = registerElement('a-mixin', {
-  prototype: Object.create(ANode.prototype, {
-    createdCallback: {
-      value: function () {
-        this.componentCache = {};
-        this.id = this.getAttribute('id');
-        this.isMixin = true;
-      }
-    },
+class AMixin extends ANode {
+  constructor () {
+    super();
```

Comment:
is ANode is extending HTMLElement, same `self = super(self)` dance might apply

Reply:
Yeah, I think you're commenting in an outdated snippet:

https://github.com/aframevr/aframe/pull/4167/files#diff-631b4510fca3a68661ae2833c57e1e3dR18

Thanks

Reply:
I did, apologies for the confusion